### ISSUE TITLE
Can't set attention_probs_dropout_prob in LEDConfig

### ISSUE THREAD

**Issue by alceausu** [huggingface/transformers#12907]
## Environment info

- `transformers` version: 4.9.0
- Python version: 3.8.10
- PyTorch version (GPU?): 1.9.0

### Who can help
@patrickvonplaten @beltagy 

## Information

Loading LEDForConditionalGeneration throws an error on line 314 of configuration_utils.py:
```
"Can't set attention_probs_dropout_prob with value 0.1 for LEDConfig"
```
but this parameter is required in line 149 of [modeling_led.py](https://github.com/huggingface/transformers/blob/master/src/transformers/models/led/modeling_led.py)
```
classLEDEncoderSelfAttention(nn.Module):
...
   self.dropout=config.attention_probs_dropout_prob
```
It works fine if I remove it from the config.

## To reproduce

I am trying to load a LEDForConditionalGeneration, converted from a Bart model.
See [convert_bart_to_longformerencoderdecoder.py](https://github.com/allenai/longformer/blob/master/scripts/convert_bart_to_longformerencoderdecoder.py) and some hints on how to [replace LongformerEncoderDecoderForConditionalGeneration with LEDForConditionalGeneration](https://github.com/allenai/longformer/issues/192)

**Comment by NielsRogge** [comment]
The reason you're getting this error is because attention_probs_dropout_prob is only defined as a getter, not as a setter, as you can see [here](https://github.com/huggingface/transformers/blob/ba15fe7995a02357ecea6e7024918f6915564c36/src/transformers/models/led/configuration_led.py#L173-L175). The reason for this is that some models call this dropout value "attention_dropout", while others call it "attention_probs_dropout_prob". 

Also relevant for @nreimers, who is currently working on adding setters (besides getters) for all attribute names.
